src/test/features/response/tasks/whenWillYouPayTask.ts

```diff
@@ -0,0 +1,35 @@
+import { expect } from 'chai'
+
+import { WhenWillYouPayTask } from 'response/tasks/whenWillYouPayTask'
+
+import { ResponseDraft } from 'response/draft/responseDraft'
+import { DefendantPaymentType, DefendantPaymentOption } from 'response/form/models/defendantPaymentOption'
+
+describe('WhenWillYouPayTask', () => {
+
+  describe('when response', () => {
+    it('should be completed when option is selected', () => {
+      const draft: ResponseDraft = new ResponseDraft()
+      draft.defendantPaymentOption = new DefendantPaymentOption().deserialize({ option: { value: DefendantPaymentType.INSTALMENTS.value } })
+      expect(WhenWillYouPayTask.isCompleted(draft)).to.equal(true)
+    })
+
+    it('should be completed when option is undefined', () => {
+      const draft: ResponseDraft = new ResponseDraft()
+      draft.defendantPaymentOption = new DefendantPaymentOption().deserialize(undefined)
+
+      expect(WhenWillYouPayTask.isCompleted(draft)).to.equal(false)
```

Comment:
Scenario says `should be completed when option is undefined` while assertion is `to.equal(false)`. Total mess!